src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -162,7 +159,7 @@ var mediaEvents = {
   topWaiting: 'waiting',
 };
 
-function trapClickOnNonInteractiveElement(inst) {
+function trapClickOnNonInteractiveElement(node : any) {
```

Comment:
can you make it Element? even if Element doesn't have onclick, nice to check that the caller is passing in the right type